sql_metadb/derived_tables/invoice_adjustments_ext.sql

```diff
@@ -17,7 +17,7 @@ SELECT
     invl.id AS invl_id,
     coalesce(jsonb_extract_path_text(invl.jsonb, 'total')::numeric(19,4), 0) AS invoice_line_value,
     fintrainvl.transaction_amount AS transaction_invoice_line_value, --This is invoice_line_value in system currency
-    coalesce(invadj.adjustment_value, 0) AS inv_adjust_total_value, 
+    coalesce(invadj.adjustment_value, 0::numeric(19,4)) AS inv_adjust_total_value, 
     fintrainv.transaction_amount AS transaction_invoice_adj_value, --This is inv_adjust_total_value in system currency
     invltotal.invl_total AS "invls_total",
```

Comment:
The fields `transaction_invoice_line_value`, `transaction_invoice_adj_value `and `invl_total `are monetary values too.